src/locales/tr-TR.po

```diff
@@ -32,23 +32,23 @@ msgstr ""
 
 #: src/pages/SwapV2/index.tsx
 msgid "+ Add Recipient (optional)"
-msgstr "+ Alıcı Ekle (isteğe bağlı)"
+msgstr ""
```

Comment:
4 files in total @XiaoYhun

Reply:
hahaha, this is a bug before by @viet, I was fixed on the main branch but not in my branch so the bug came back.

Reply:
lol please go sleep :tien-frown: